src/style.css

```
@@ -1,3 +1,62 @@
+:root {
+  --blue-color: #00acdc;
```

```suggestion
  --active-link-color: #00acdc;
```
It's better to name variables to show their purpose because color could be another in the future